lib/xenops_server.ml

```diff
@@ -1009,7 +1012,9 @@ let rec atomics_of_operation = function
            (VBD_DB.vbds id)
         ) @ (List.map (fun vif -> VIF_set_active (vif.Vif.id, false))
                (VIF_DB.vifs id)
-            ) @ [
+               ) @ (List.map (fun vgpu -> VGPU_set_active (vgpu.Vgpu.id, false))
+                      (VGPU_DB.vgpus id)
+                   ) @ [
       VM_hook_script(id, Xenops_hooks.VM_post_destroy, reason)
     ]
   | VM_reboot (id, timeout) ->
```

Comment:
During a reboot this will keep the vgpu status "active", right?

Reply:
Yes, that is one of the objectives.

Reply:
Yes @edwintorok .